src/main/java/uk/gov/homeoffice/digital/sas/timecard/model/TimePeriodType.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.homeoffice.digital.sas.timecard.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import uk.gov.homeoffice.digital.sas.jparest.annotation.Resource;
+import uk.gov.homeoffice.digital.sas.jparest.models.BaseEntity;
+
+import javax.persistence.Entity;
+import javax.validation.constraints.NotEmpty;
+
+@Resource(path = "time-period-type")
+@Entity(name = "time_period_type")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@NoArgsConstructor
+@Getter
+@Setter
+public class TimePeriodType extends BaseEntity {
```

Comment:
Good point from @daleelR here: https://homeofficedigital.slack.com/archives/C02JB7AL73N/p1661341348832679

Could we disable all the methods apart from get for TimePeriodType?

Reply:
Ah Suresh has clarified that jparest does not currently offer the functionality to disable particular methods for a particular entity